code/controllers/configuration.dm

```diff
@@ -166,6 +166,7 @@
 	var/mommi_static = 0 //Scrambling mobs for mommis or not
 
 	var/skip_minimap_generation = 0 //If 1, don't generate minimaps
+	var/skip_holominimap_generation = 0 //If 1, don't generate holominimaps
```

Comment:
did you intend to put this in this PR?

Reply:
No I committed it by accident
but I don't really want to revert it and move it to another PR now